crm_claim_rma/models/crm_claim.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright 2015 Eezee-It, MONK Software, Vauxoo
+#    Copyright 2013 Camptocamp
+#    Copyright 2009-2013 Akretion,
+#    Author: Emmanuel Samyn, Raphaël Valyi, Sébastien Beau,
+#            Benoît Guillot, Joel Grand-Guillaume, Leonardo Donelli,
+#            Osval Reyes, Yanina Aular
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import _, api, exceptions, fields, models
+
+from .invoice_no_date import InvoiceNoDate
+from .product_no_supplier import ProductNoSupplier
+
+
+# TODO add the option to split the claim_line in order to manage the same
+# product separately
+class CrmClaim(models.Model):
+    _inherit = 'crm.claim'
+
+    def _get_default_warehouse(self):
+        company_id = self.env.user.company_id.id
+        wh_obj = self.env['stock.warehouse']
+        wh = wh_obj.search([('company_id', '=', company_id)], limit=1)
+        if not wh:
+            raise exceptions.Warning(
+                _('There is no warehouse for the current user\'s company.'))
+        return wh
+
+    @api.multi
+    def name_get(self):
+        res = []
+        for claim in self:
+            code = claim.code and str(claim.code) or ''
+            res.append((claim.id, '[' + code + '] ' + claim.name))
+        return res
+
+    company_id = fields.Many2one(
+        change_default=True,
+        default=lambda self: self.env['res.company']._company_default_get(
+            'crm.claim'))
+
+    claim_line_ids = fields.One2many('claim.line', 'claim_id',
+                                     string='Return lines')
+    planned_revenue = fields.Float('Expected revenue')
+    planned_cost = fields.Float('Expected cost')
+    real_revenue = fields.Float()
+    real_cost = fields.Float()
+    invoice_ids = fields.One2many(
+        'account.invoice',
```

Comment:
You don't need to put one argument per line. You can join several arguments in the same line if they fits in 80 cols.